crates/bootstrap_srv/src/lib.rs

```diff
@@ -45,6 +45,16 @@
 ///
 /// Any other properties on these objects will be ignored and pass through.
 ///
+/// A note on the "isTombstone" field: This is named "tombstone" because the
+/// p2p gossip this bootstrap service is designed to facilitate must keep
```

Comment:
```suggestion
/// p2p gossip this bootstrap service must keep
```

Reply:
Attempted to fix this via: https://github.com/holochain/kitsune2/pull/57/commits/6e482e3d231d14b62e108016c1240bfb9c2400e3, let me know if that works.